images/devtools-golang-v1beta1/context/Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM docker.io/safewaters/docker-lock:latest@sha256:432d90ddc2891f4845241adc63e5eef2dd1486fa14ea7882433cbd3f8ed64622 AS docker-lock
-FROM golangci/golangci-lint:v1.48.0-alpine@sha256:e9ce901b7974d641d6288b070fc619d199a823d0f696563fec8be90f19789b57 AS golangci-lint
+FROM golangci/golangci-lint:v1.50-alpine@sha256:a392d4e44049a444a927878792dae9534075ec57880e0657647ce818bd8278c2 AS golangci-lint
 FROM docker.io/mikefarah/yq:4@sha256:d4e1d060e248bdc3ac9e0fccfb4209e24ff42ed3d39beab33d3c3a19480b14fd AS yq
 FROM docker.io/hadolint/hadolint:v2.12.0@sha256:30a8fd2e785ab6176eed53f74769e04f125afb2f74a6c52aef7d463583b6d45e AS hadolint
 FROM --platform=linux/amd64 docker.io/goodwithtech/dockle:v0.4.10@sha256:26524f9f4d1750be7316870f2e74066305b1747a16b1584e522a136b2d507af1 AS dockle
 FROM docker.io/moby/buildkit:v0.10.6-rootless@sha256:df14db059c966a3329b223f32bb6de6f91711c8f96090e71036fffa983d76eed AS buildkit
-FROM gcr.io/go-containerregistry/crane:v0.12.1@sha256:2049f5fa913070ffb0f093ee0f4251f8cb9a142d601c84f89ef88593f82c7425 AS crane
+FROM gcr.io/go-containerregistry/crane:latest@sha256:ee086bf7941ac29fcf9ea6e70f15bd634a5ff89d900bcd27d7bfb557806089e3 AS crane
```

Comment:
Why do you remove the actual tag version and use the `latest` tag instead?

I am just wondering, obviously a digest determines the actual image version.

Reply:
I thought they stopped tagging releases as the `latest` tag is quite recent and the `v0.12.1` tag is quite old, but I see now they just tag things to `latest` that is not released.

I will revert back to using tagged. Thanks for highlighting it.

Reply:
actually to be fair, I'm not entirely sure what is happening with the tags, but it is weird enough that the safe bet is probably to stick to using v0.12.1